events/omero-workshop-eastbio-dundee-november-2022.html

```diff
@@ -0,0 +1,49 @@
+---
+layout: news-events
+filename: events
+title: November 2022 OMERO EastBio workshop
+main-blurb: 1-hour online demo/workshop on how to use OMERO.
+---
+     <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">The OMERO demo/workshop is a part of a whole-day course for Postdoctoral Students.</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Organiser: <a href="http://www.eastscotbiodtp.ac.uk/" target="_blank">EastBio</a></h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Location: Dundee, UK</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Date: 22nd November 2022</h2>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Duration: approx 1 hour</h2>
+                <p class="event-day-description">The Open Microscopy Environment (OME) is an open-source software project that develops tools that enable access, analysis, visualization, sharing and publication of biological image data. We support more than 150 image data formats across many imaging modalities including medical imaging, high-content screening and whole-slide imaging.
+                We support and visualize multi-channel 3D timelapse images.
+ 
+                OMERO, a component of OME, is a software platform for image data management and analysis which is presented in this course. 
+                Import, organisation, viewing, searching, annotation and publishing of imaging data with OMERO is shown.
+                </p>
+                <p class="event-day-description">
+                The workshop concentrates on basics of OMERO and did not assume any programming skills.
+                </p>
+            </div>
+        </div>
+        <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Resources will be available under: <a href="https://downloads.openmicroscopy.org/presentations/2022/Eastbio/" target="_blank">presentation and walkthrough</a>
```

Comment:
I think we can remove "will be" otherwise we will have to update the entry and we will probably forget

Reply:
Fixed in https://github.com/ome/www.openmicroscopy.org/pull/610/commits/52a3262825512ba1588e7ae0f099cf90dc34e113